s4e-backend/src/test/java/pl/cyfronet/s4e/data/repository/query/QueryBuilderTest.java

```diff
@@ -34,7 +34,7 @@ public void shouldReturnWholeQuery() {
         params.put("ingestionFrom", "2019-11-09T00:00:00.000000+00:00");
         params.put("ingestionTo", "2019-11-12T00:00:00.000000+00:00");
         StringBuilder resultQuery = new StringBuilder();
-        String query = "SELECT id,product_id,timestamp  FROM Scene WHERE true  " +
+        String query = "SELECT id,product_id,ST_AsText(footprint) AS footprint,metadata_content,scene_content,timestamp  FROM Scene WHERE true  " +
```

Comment:
Why you're not using `stringBuilder` and `string formatting` to build this query?